hermes_eea/calibration/build_spectra.py

```diff
@@ -0,0 +1,57 @@
+
+from hermes_core.timedata import HermesData
+import astropy.units as astropy_units
+from astropy.timeseries import TimeSeries
+from astropy.time import Time
+from hermes_core.timedata import HermesData
+from astropy.nddata import NDData
+from ndcube import NDCube, NDCollection
+from hermes_eea.util.time.iso_epoch import epoch_to_iso_obj, epoch_to_eea_iso, epoch_to_iso
+import numpy as np
+from astropy.wcs import WCS
+
+class Build_Hermes_EEA_Data:
+
+    def __init__(self, myEEA):
+        self.EEA = myEEA
+        self.raw_counts = astropy_units.def_unit("raw instrument counts")
```

Comment:
Please consider using an actual unit. There are a number which might be more appropriate (like just counts for example).

Reply:
this raw instrument counts thing is something we use in FPI. When we are finalizing our model I'l mention this.